src/App.js

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useState } from "react";
 import FlashCardList from "./Components/FlashCardList";
+import Header2 from "./Components/Header2";
 import Header from "./Components/Header";
-import GlobeLogo from "./Components/GlobeLogo";
```

Comment:
Move the svg in Header to a separate component and reuse that in other components